scripts/package_test.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Test installation and use of the ncpi-fhir-utility package
+
+set -eo pipefail
+
+echo "Begin package install and usage test ..."
+
+# Setup
+echo "Setup python venv"
+mkdir workspace
+cd workspace
+cp -r ../tests .
+
+python3 -m venv venv
+source ./venv/bin/activate
+pip install --upgrade pip
+
+# If not running on CircleCI get the branch we're on
+if [[ -z $CIRCLE_BRANCH ]];
+then
+    CIRCLE_BRANCH=$(git branch | sed -n -e 's/^\* \(.*\)/\1/p')
+else
+    RUN_LOCAL="1"
+fi
+
+# Test
+echo "Install ncpi_fhir_utility, using branch $CIRCLE_BRANCH"
+echo "ncpi_fhir_utility @ git+https://git@github.com/ncpi-fhir/ncpi-fhir-utility.git@$CIRCLE_BRANCH" \
+    >> requirements.txt
+pip install -r requirements.txt
+
+cp tests/data/profiles/valid/StructureDefinition-participant.json \
+tests/data/site_root/input/resources/profiles
+fhirutil validate tests/data/site_root/ig.ini --publisher_opts="-tx n/a" --clear_output
+
+# Clean up & reset
```

Comment:
I know this is a little weird, but if I don't do it this way, there are a number of other changes I'd have to make to all of: Dockerfile, run_publisher.sh, and this script - which I don't think are worth it